grammar/argextend.mlp

```diff
@@ -11,8 +11,14 @@ open Q_util
 let loc = Ploc.dummy
 let default_loc = <:expr< Loc.ghost >>
 
+IFDEF STRICT THEN
+  let ploc_vala x = Ploc.VaVal x
+ELSE
+  let ploc_vala x = x
+END
+
```

Comment:
@ppedrot In fact, it is not easy to put it in `q_util.mlp`, since it has an interface and the type of this function changes according to the mode...